docs/source/user/subdyn/theory.rst

```diff
@@ -391,18 +391,18 @@ element stiffness and consistent mass matrices can be written as follows
-                                                                           &                                                  &                   &                                                                        &                                                                        &                   &                                                    &                                                     & k_{33}             & 0                                                                     & 0                                                                     & 0 \\
-                                                                           &                                                  &                   &                                                                        &                                                                        &                   &                                                    &                                                     &                    & k_{44}                                                                & 0                                                                     & 0   \\
-                                                                           &                                                  &                   &                                                                        &                                                                        &                   &                                                    &                                                     &                    &                                                                       & k_{55}                                                                & 0   \\
-                                                                           &                                                  &                   &                                                                        &                                                                        &                   &                                                    &                                                     &                    &                                                                       &                                                                       & k_{66} \\
+                         \frac{12 E J_y} {L_e^3 \left(  1+ K_{sy} \right)} & 0                                                & 0                 & 0                                                                      & \frac{6 E J_y}{L_e^2 \left(  1+ K_{sy} \right)}                        & 0                 & -\frac{12 E J_y}{L_e^3 \left(  1+ K_{sy} \right)}  & 0                                                   & 0                  & 0                                                                      & \frac{6 E J_y}{L_e^2 \left(  1+ K_{sy} \right)}                        & 0 \\
```

Comment:
Could you also remove the `\setcounter{MaxMatrixCols}{20}` from line 389.  It doesn't appear to do anything and renders really oddly: https://openfast.readthedocs.io/en/dev/source/user/subdyn/theory.html#beam-element-formulation